oauth2/oauth2-client/src/test/resources/oauth2-client-schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE oauth2_authorized_client (
```

Comment:
Can we publish this on the classpath similar to how we do with the users.ddl? We could also update the Javadoc of the service to mention the classpath resource.